tests/test_core.py

```diff
@@ -1116,6 +1116,17 @@ def test_exceptions(self):
       self.set_setting('SUPPORT_LONGJMP', support_longjmp)
       self.do_run_from_file(test_file('core', 'test_exceptions.cpp'), test_file('core', 'test_exceptions_caught.out'))
 
+  # We have LTO tests covered in 'wasmltoN' targets, but they don't run as a
+  # part of Emscripten CI, so we make a separate LTO test here.
+  @with_both_exception_handling
+  def test_exceptions_lto(self):
+    self.set_setting('EXCEPTION_DEBUG')
+    self.emcc_args += ['-flto']
+    self.maybe_closure()
+    for support_longjmp in [0, 1]:
```

Comment:
Can you do this with:

```
  @parameterized({                                                                                                   
    '': [False],                                                          
    'longjmp': [True],                                                      
  })     
```

.. so we get two different test here.

Reply:
There are many (but not all) EH tests that contain this pattern. Can we do this as a followup?

Reply:
Sure no worries.  I'm just trying to reduce the number of tests that contain loops like this that make debugging harder so I keep an eye out for them when I see them.

Reply:
I looked into this, but I'm not sure if the current framework supports nested parameterization.

I thought there were many EH tests that contained this pattern, but it turned out currently only two of them had this pattern. One is [`test_exceptions`](https://github.com/emscripten-core/emscripten/blob/eb3f75206c0a160a2774ae4f983e1c866745c94c/tests/test_core.py#L1111-L1117), decorated with [`with_both_exception_handling`](https://github.com/emscripten-core/emscripten/blob/eb3f75206c0a160a2774ae4f983e1c866745c94c/tests/test_core.py#L90-L112), and the other is [`test_exceptions_off`](https://github.com/emscripten-core/emscripten/blob/eb3f75206c0a160a2774ae4f983e1c866745c94c/tests/test_core.py#L1119-L1122), which is not decorated with anything.

We have functions that are decorated with more than one decorator, but we don't currently have test functions with two decorators, each of which has its own parameterization. I tried both
```py
  @parameterized({
    '': (False,),
    'longjmp': (True,)
  })
  @with_both_exception_handling 
```
and
```py
  @with_both_exception_handling 
  @parameterized({
    '': (False,),
    'longjmp': (True,)
  })
```
but the resulting created functions only seems to recognize the outer one. Do you have any ideas on how to make this work?

But also I don't feel strongly about the necessity of this because there are only two functions that use this anyway.

Reply:
Sure.. I can imagine that might not work.. or be tricky.  Certainly not urgent to address.